styles/pages/Home.module.scss

```diff
@@ -1,446 +1,447 @@
 @import "styles/variables.scss";
 
 @-webkit-keyframes fadeEffect {
-	from {
```

Comment:
Just to check you didn't modify this file right, it was auto-formatted? If so, could you undo this and use the old version? I'll standardise the formatting soon haha.